src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.js

```diff
@@ -337,7 +339,7 @@ describe('VAOS Page: ConfirmedAppointmentDetailsPage with VAOS service', () => {
         name: 'You shared these details about your concern',
       }),
     ).to.be.ok;
-    expect(screen.getByText(/New medical issue: I have a headache/)).to.be.ok;
+    expect(screen.getByText(/New medical issue/)).to.be.ok;
```

Comment:
Just checking that we're intentionally changing the behavior being tested here? I think it does make sense to test the case where coding is present but text (i.e. what's returned by `additionalAppointmentDetails)` is missing.

Reply:
Agreed!